README.md

```diff
@@ -2,65 +2,60 @@
 Simple data loader for CGP HCA Data Store
 
 ## Common Setup
-1. Clone the repo:
+1. **(optional)**  We recommend using a Python 3
+   [virtual environment](https://docs.python.org/3/tutorial/venv.html).
 
-   `git clone https://github.com/DataBiosphere/cgp-dss-data-loader.git`
+1. Run:
 
-2. Go to the root directory of the cloned project:
-   
-   `cd cgp-dss-data-loader`
+   `pip3 install cgp-dss-data-loader`
 
-3. Run (ideally in a new [virtual environment](https://docs.python.org/3/tutorial/venv.html)):
+## setup for development
```

Comment:
Is there a reason that all of this is lowercase now?

Reply:
whoops!